air-script/tests/aux_trace/aux_trace.masm

```diff
@@ -45,6 +62,16 @@ proc.compute_integrity_constraint_divisor
     # => [divisor_1, divisor_0, ...]
 end # END PROC compute_integrity_constraint_divisor
 
+# Procedure to evaluate each integrity constraints.
+#
+# All the 3 main and 2 auxiliary constraints are computed.
+# The result of each evaluation is kept in the stack, with the top of the stack
+# containing the evaluations for the auxiliary trace (if any) followed by the main trace.
+#
+# Input: [...]
+# Output: [(r_1, r_0)*, ...]
+# where: (r_1, r_0) is the quadratic extension resulting from the integrity constraint evaluation.
```

Comment:
nit: should probably say "quadratic extension element" rather than just "quadratic extension".

Reply:
changed